va/va.h

```diff
@@ -392,7 +392,8 @@ typedef enum
     VAProfileHEVCMain444_12             = 28,
     VAProfileHEVCSccMain                = 29,
     VAProfileHEVCSccMain10              = 30,
-    VAProfileHEVCSccMain444             = 31
+    VAProfileHEVCSccMain444             = 31,
```

Comment:
Do we support no scc(nature content) in scc profile?
If yes, since we have palette_mode_enabled_flag to indicate scc enabled or not, whey we need a new scc profile?
If no, how we support content switch, from nature content to a screen content.

Reply:
Thanks for clarification from cwang105.  SCC profiles is back compatible, can support the preceding profiles.